src/tsickle.ts

```diff
@@ -1163,22 +1167,30 @@ class Annotator extends ClosureRewriter {
     const importPath = this.resolveModuleSpecifier(specifier);
     const nsImport = es5processor.extractGoogNamespaceImport(importPath);
     const forwardDeclarePrefix = `tsickle_forward_declare_${++this.forwardDeclareCounter}`;
-    const moduleNamespace =
-        nsImport !== null ? nsImport : this.host.pathToModuleName(this.file.fileName, importPath);
+
     const moduleSymbol = this.typeChecker.getSymbolAtLocation(specifier);
     // Scripts do not have a symbol. Scripts can still be imported, either as side effect imports or
     // with an empty import set ("{}"). TypeScript does not emit a runtime load for an import with
     // an empty list of symbols, but the import forces any global declarations from the library to
     // be visible, which is what users use this for. No symbols from the script need forward
     // declaration, so just return.
     if (!moduleSymbol) return;
-    const exports = this.typeChecker.getExportsOfModule(moduleSymbol);
+
+    // Convert the source fileName to a logical fileName.
+    const rootDir = (this.tsOpts && this.tsOpts.rootDir) || '';
```

Comment:
Is rootDir guaranteed to have a trailing slash?  I think maybe you want path.relative() here.

Reply:
Done.